Library/Homebrew/extend/os/mac/utils/bottles.rb

```diff
@@ -42,6 +42,8 @@ def find_older_compatible_tag(tag)
         return if tag_version.blank?
 
         keys.find do |key|
+          # Not sure about this?
+          next if key.nil?
```

Comment:
What's this doing/for?

Reply:
It's not needed, it was due to a bug I introduced myself. Change removed from the PR.